backend/graminsta/graminsta/settings.py

```diff
@@ -25,6 +25,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
+# Root path for resources uploaded
+MEDIA_ROOT = "img/"
```

Comment:
As there are multiple types of media (image, text, video, audio), use `media/` as `MEDIA_ROOT` might be better?